src/main/lua/exasolvs/RemoteQueryRewriter.lua

```diff
@@ -0,0 +1,48 @@
+local AbstractQueryRewriter = require("exasolvs.AbstractQueryRewriter")
+local QueryRenderer = require("exasolvs.QueryRenderer")
+local ImportBuilder = require("exasolvs.ImportBuilder")
+
+--- This class rewrites the query.
+-- @classmod RemoteQueryRewriter
+local RemoteQueryRewriter = {_NAME = "RemoteQueryRewriter"}
+RemoteQueryRewriter.__index = RemoteQueryRewriter
+setmetatable(RemoteQueryRewriter, {__index = AbstractQueryRewriter})
+
+--- Create a new instance of a <code>RemoteQueryRewriter</code>.
+-- @param connection_id name of the connection object that defines the details of the connection to the remote Exasol
```

Comment:
```suggestion
-- @param connection_id name of the connection object that defines the details of the connection to the remote Exasol
```
Is this the connection **id** or a connection **object**? Either rename the parameter or update the comment.